src/server/api/routers/auth.ts

```
@@ -104,4 +108,132 @@ export const authRouter = createTRPCRouter({
+        family_name?: string;
+      };
+
+      if (!tokenInfo.email) {
+        throw new TRPCError({ code: 'UNAUTHORIZED', message: 'Invalid Google ID token' });
+      }
+
+      const normalizedEmail = tokenInfo.email.toLowerCase();
+
+      const user = await db.user.findUnique({
+        where: { email: normalizedEmail },
+        select: { id: true, email: true },
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'select' should be before 'where'.

(sort-keys-fix/sort-keys-fix)